lib/JMAP/TestSuite/Comparator/Calendar.pm

```diff
@@ -0,0 +1,47 @@
+package JMAP::TestSuite::Comparator::Calendar;
+use Moose;
+
+use Test::Deep ':v1';
+use Test::Deep::JType;
+use Test::Deep::HashRec;
+
+use Sub::Exporter -setup => [ qw(calendar) ];
+
+sub calendar {
+  my ($overrides) = @_;
+
+  $overrides ||= {};
+
+  my %required = (
+    id              => jstr,
+    name            => jstr,
+    color           => jstr,
+    sortOrder       => jnum,
+    isVisible       => jbool,
```

Comment:
The following non-null, non-default properties are missing:
- isSubscribed
- isDefault
- includeInAvailability